manta.wdl

```diff
@@ -0,0 +1,63 @@
+version 1.0
+
+task Somatic {
```

Comment:
Why is task called Somatic while the tool is called manta?

Reply:
Manta has two possible configurations, somatic and germline. I didn't implement the option for germline SV calling, but as that might be implemented in the future I thought naming this somatic would be more appropriate (like how the BWA mapping task is called 'Mem' and not 'Bwa').

Reply:
:+1: